src/commons/repl/Repl.tsx

```diff
@@ -81,6 +86,16 @@ export const Output: React.FC<OutputProps> = (props: OutputProps) => {
             <Pre className="logOutput">Check out the Stepper tab!</Pre>
           </Card>
         );
+      } else if (props.isHtml) {
+        return (
+          <Card>
+            <Pre className="logOutput">
+              Check out the HTML Display tab!
+              <br />
+              Please check the browser console for more detailed errors and warnings.
```

Comment:
A possible concern is that someone who might want to use the HTML display is probably a beginner and thus unfamiliar with the developer tools, in which case this statement makes no sense to them.

Reply:
The reason why the line is there is due to browser differences, e.g. Chrome adds extra warnings, Safari doesn't allow routing of console errors so the error message displayed will always be "Line 0: Script error".

Maybe it would be better for this explanation to be on the Playground Home tab, I'll shift the information there instead to avoid the output display being overly cluttered.